### PR TITLE
core/issues/96 - Expose source column in booking report

### PR DESCRIPTION
**Pull request by yashodha**
Overview
----------------------------------------
Expose source column in booking report

Before
----------------------------------------
book_before

After
----------------------------------------

book_after

### CODE REVIEW

File: CRM/Report/Form/Contribute/Bookkeeping.php [hunk 1]
```diff
@@ -301,6 +301,9 @@ public function __construct() {
             'title' => ts('Contribution Status'),
             'default' => TRUE,
           ),
+          'source' => array(
```
Comment (eileenmcnaughton) [inline CRM/Report/Form/Contribute/Bookkeeping.php]: Wouldn't it be better to use the same key as below & define a 'name' as below?

**Comment by eileenmcnaughton** [comment]
Makes sense - one minor comment inline

**Comment by eileenmcnaughton** [comment]
@yashodha @nganivet just a minor thing to get this mergeable - but I notice there are a few PRs you have raised that have unanswered reviewer feedback blocking them being merged. We should figure out which ones you want to follow up on & close the others

**Comment by mattwire** [comment]
@yashodha Could you update the PR subject per https://docs.civicrm.org/dev/en/latest/tools/git/#pr-subject:
"dev/core#96 - Expose source column in booking report"

**Comment by eileenmcnaughton** [comment]
@yashodha @nganivet I'm going to close this & let you re-open it when the reviewer feedback is addressed. There is nothing lost by closing a PR as it can easily be re-opened and there is a real time cost in reviewers check in on a PR only to find it is not reviewable (e.g because a change needs to be made or the PR requires a unit test).

If you need a way to track changes you are working on getting upstreamed & you want to use something other than your own issue tracker then gitlab is a good option

**Pull request closed without merging**